### ISSUE TITLE
99th_percentile_frame_rasterizer_time_millis needs a test for a full shader cache

### ISSUE THREAD

**Issue by gaaclarke** [flutter/flutter#68367]
After https://github.com/flutter/flutter/pull/68128 was landed the flutter_gallery is uninstalled between tests.  This has the affect that the shader cache is thrown out.  Now `99th_percentile_frame_rasterizer_time_millis` is showing a reading for an app without a full shader cache.  It would be nice if we could tweak the test so that we can measure both values, once with the cache full and once empty.

related issue: https://github.com/flutter/flutter/issues/68316

cc @liyuqian @jonahwilliams

**Comment by liyuqian** [comment]
I agree that the regression #68316 is caused by some caching that gets deleted during an uninstall. However, I doubt whether it's the shader persistent cache because it only helps significantly on Android, but not on iOS yet. @gaaclarke : can you please run the test multiple times with `--purge-persistent-cache` flag which will remove all shader cache without uninstalling the app? I expect it to be fast, and thus rule out the persistent cache to be the problem here.

**Comment by gaaclarke** [comment]
I don't think I'll be able to get to it today because of sheriffing, but no matter what the cause is I think we should track these 2 numbers separately (performance on first launch, performance on subsequent launches).  There could potentially be a regression in one and not the other that we'd want to capture.

**Comment by liyuqian** [comment]
Cool, I just revised the issue title to reflect that.
